inc/plugins/myalerts.php

```diff
@@ -1206,6 +1207,56 @@ function myalerts_alert_rated_threadauthor()
     }
 }
 
+$plugins->add_hook(
+	'polls_vote_end',
+	'myalerts_alert_voted_threadauthor'
+);
+function myalerts_alert_voted_threadauthor()
+{
+	global $mybb, $db, $poll;
+
+    if (!isset($mybb->user['uid']) || $mybb->user['uid'] < 1) {
+        return;
+    }
+
+    myalerts_create_instances();
+
+    $alertTypeManager = MybbStuff_MyAlerts_AlertTypeManager::getInstance();
+
+    if (is_null($alertTypeManager) || $alertTypeManager === null) {
```

Comment:
`getInstance()` can return `false` (I know, stupid design decision - it needs changing) and as such `false` must be checked for here.